addons/web_editor/static/src/js/editor/image_processing.js

```diff
@@ -430,18 +430,9 @@ async function loadImageInfo(img, rpc, attachmentSrc = '') {
     if (img.dataset.originalSrc || !src) {
         return;
     }
-    // In order to be robust to absolute, relative and protocol relative URLs,
-    // the src of the img is first converted to an URL object. To do so, the URL
-    // of the document in which the img is located is used as a base to build
-    // the URL object if the src of the img is a relative or protocol relative
-    // URL. The original attachment linked to the img is then retrieved thanks
-    // to the path of the built URL object.
-    const srcUrl = new URL(src, img.ownerDocument.defaultView.location.href);
-    const relativeSrc = srcUrl.pathname;
-
     const {original} = await rpc({
         route: '/web_editor/get_image_info',
-        params: {src: relativeSrc},
+        params: {src: src, context: {base: img.ownerDocument.defaultView.location.href}},
```

Comment:
Would this not work:
```suggestion
        params: {src: src},
        context: {base: img.ownerDocument.defaultView.location.href},
```
?
Would be prettier (even if it does the same in the end).